README.md

```diff
@@ -1,4 +1,35 @@
 # Newoffers Messenger
 
-Newoffers Messenger is a Pub/Sub messaging provider using aws SNS
+Newoffers Messenger is a Pub/Sub messaging provider using AWS SNS os Azure ServiceBus
```

Comment:
Newoffers Messenger is a Pub/Sub messaging provider using AWS SNS or Azure ServiceBus